Compiler/compiler.cpp

```diff
@@ -28,7 +28,7 @@ void __cdecl DolphinTrace(LPCTSTR format, ...)
 	char buf[1024];
```

Comment:
While fixing this, may as well fix the error in the original code:-). This buffer should be TCHAR since the format arg is of that form. ....